docs/gathered-data.md

```diff
@@ -885,6 +885,17 @@ gathers the alerts that have been silenced.
   * 4.10+
 
 
+## SupportSecret
+
+gathers anonymized support secret if there is any
+
+* Location in archive: config/secrets/openshift-config/support/data.json
+    (can be ommited if the secret doesn't exist)
+* Id in config: support_secret
+* Since version:
+  * 4.12+
```

Comment:
I think we are about to merge it so please change to `4.11+`

Reply:
done